static/src/javascripts/projects/commercial/modules/prebid/utils.js

```diff
@@ -12,8 +12,19 @@ import type { PrebidSize } from './types';
 const isInAppnexusUSAdapterTestVariant = (): boolean =>
     isInVariantSynchronous(appnexusUSAdapter, 'variant');
 
+const SUFFIX_REGEXPS = {};
```

Comment:
I've stared at this for a bit and remain confused. Is this a const object that is expected to be mutated? If so, is that a sensible implementation?

Reply:
I'm far from being a JS idioms doctor, so from the purely technical POV, the container (the object {}) is constant, but its content can change. This implementation avoids building the same RegExps over and over again for all the slots in the page ( <N of GTP Slots> x <N of potential Prebid slot matches> ). SUFFIX_REGEXPS acts as a cache for these RegExps. See http://jsben.ch/VcR8B